### ISSUE TITLE
No Update Required on any Admin Request

### ISSUE THREAD

**Issue by sosensible** [wpengine/hgv#232]
I am glad no update is required, but I cannot get into the admin section of the HVM or PHP site. It does appear to log in correctly though. I was running an earlier version and just did a full redo...

vagrant destroy
vagrant up

Then I did a destroy, deleted the hvm folder, pulled a git clone, and reset things... and it works.

**Comment by ericmann** [comment]
This is a persistent issue with WordPress itself on certain updates. The field that tracks the installed DB version is sometimes cached and will trigger an attempt for WordPress to update the database. Unfortunately, the database is _already_ updated and the admin will end up in an endless loop of checking for a required update.

It's annoying, but usually resolves itself after a few minutes (or after turning the machine off and on again). Destroying and reprovisioning is the nuclear option, but will also resolve it as you've already discovered.

In short, this is a known issue with Core itself, not something that's unique to HGV. If you want to track the Core issue, you can do so on Trac: https://core.trac.wordpress.org/ticket/27669

**Comment by zamoose** [comment]
`wp cache flush` also usually does the trick for me.  

--
Doug Stewart
